test/.eslintrc

```diff
@@ -1,3 +1,7 @@
 ---
 env:
   mocha: true
+rules:
+  no-param-reassign:
+    - 2
+    - props: false
```

Comment:
I think this change was to allow you to change the `record.hasTimestamps` later. If so, please check https://github.com/opentrials/api/pull/45/files#r63379124. Maybe this isn't needed after all.

It's OK to change the eslint rules when they don't make sense, but try to avoid changing globally. You can ignore rules for specific lines or files. For example, check https://github.com/opentrials/opentrials/blob/feature/contribute-data/models/base.js#L13